src/parser-postcss.js

```diff
@@ -206,20 +203,21 @@ function parseWithParser(parser, text) {
   return parsedResult;
 }
 
+function requireParser(isSCSS) {
+  if (isSCSS) {
+    return require("postcss-scss");
+  } else {
+    return require("postcss-less");
+  }
+}
+
 function parse(text) {
-  const r = require;
   const isLikelySCSS = !!text.match(/(\w\s*: [^}:]+|#){/);
   try {
-    return parseWithParser(
-      r(isLikelySCSS ? "postcss-scss" : "postcss-less"),
-      text
-    );
+    return parseWithParser(requireParser(isLikelySCSS), text);
   } catch (e) {
     try {
-      return parseWithParser(
-        r(isLikelySCSS ? "postcss-less" : "postcss-scss"),
-        text
-      );
+      return parseWithParser(requireParser(!isLikelySCSS), text);
```

Comment:
It's fine. We first want to check with scss (the one above) and if it doesn't work, we want to check with less (this one)